openmdao/visualization/meta_model_viewer/meta_model_visualization.py

```diff
@@ -0,0 +1,553 @@
+"""Define output of Meta Models and visualize the results."""
```

Comment:
When you made your initial branch, you should have made it on your repository, not the OpenMDAO one.  (This PR comes from OpenMDAO/mybug1.)  I think for continuity, you should close the PR and resubmit it by first pushing the branch up to your repo, then performing a PR from there.  But first, address the changes here so we don't lose them.

Going to pull down and play with the viewer, so i may have further comments.